src/patternfly/components/OverflowMenu/overflow-menu.scss

```diff
@@ -0,0 +1,141 @@
+$pf-c-overflow-menu--breakpoint-map: build-breakpoint-map("base", "md", "lg", "xl");
+
+.pf-c-overflow-menu {
+  // Base spacer - shared value
+  --pf-c-overflow-menu--spacer--base: var(--pf-global--spacer--md);
+
+  // Spacer values
+  --pf-c-overflow-menu__group--spacer: var(--pf-c-overflow-menu--spacer--base);
+  --pf-c-overflow-menu__item--spacer: var(--pf-c-overflow-menu--spacer--base);
+  --pf-c-overflow-menu__control--spacer: var(--pf-global--spacer--sm);
+
+  // Display variables
+  --pf-c-overflow-menu__group--Display: none;
+  --pf-c-overflow-menu__item--Display: none;
+
+  // Visibility variables
+  --pf-c-overflow-menu__group--Visibility: hidden;
+  --pf-c-overflow-menu__item--Visibility: hidden;
+
+  // Button group
+  --pf-c-overflow-menu__group--m-button-group--spacer: var(--pf-c-overflow-menu--spacer--base);
+  --pf-c-overflow-menu__group--m-button-group--space-items: var(--pf-global--spacer--sm);
+
+  // Icon button group
+  --pf-c-overflow-menu__group--m-icon-button-group--spacer: var(--pf-c-overflow-menu--spacer--base);
+  --pf-c-overflow-menu__group--m-icon-button-group--space-items: 0;
+
+  // Controller
+  --pf-c-overflow-menu__control--Display: inline-block;
+  --pf-c-overflow-menu__control--Visibility: visible;
+
+  display: inline-flex;
+  align-items: center;
+}
+
+// Group
+.pf-c-overflow-menu__group {
+  --pf-c-overflow-menu--spacer: var(--pf-c-overflow-menu__group--spacer);
+
+  display: var(--pf-c-overflow-menu__group--Display);
+  align-items: center;
+  visibility: var(--pf-c-overflow-menu__group--Visibility);
+
+  // Button group
+  &.pf-m-button-group {
+    --pf-c-overflow-menu--spacer: var(--pf-c-overflow-menu__group--m-button-group--spacer);
+
+    > * {
+      --pf-c-overflow-menu--spacer: var(--pf-c-overflow-menu__group--m-button-group--space-items);
+    }
+  }
+
+  // Icon button group
+  &.pf-m-icon-button-group {
+    --pf-c-overflow-menu--spacer: var(--pf-c-overflow-menu__group--m-icon-button-group--spacer);
+
+    > * {
+      --pf-c-overflow-menu--spacer: var(--pf-c-overflow-menu__group--m-icon-button-group--space-items);
+    }
+  }
+
+  &.pf-m-persistent {
```

Comment:
Persistent might be a bit arbitrary, does it mean that the items display as visible on all breakpoints?

Reply:
Yes